start-site/src/main/java/io/spring/start/site/extension/dependency/springazure/SpringAzureHelpDocumentCustomizer.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2012-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.spring.start.site.extension.dependency.springazure;
+
+import io.spring.initializr.generator.buildsystem.Build;
+import io.spring.initializr.generator.spring.documentation.HelpDocument;
+import io.spring.initializr.generator.spring.documentation.HelpDocumentCustomizer;
+
+/**
+ * A {@link HelpDocumentCustomizer} that Getting Started Guides.
+ *
+ * @author Yonghui Ye
+ */
+class SpringAzureHelpDocumentCustomizer implements HelpDocumentCustomizer {
```

Comment:
I'm not sure that this class is needed. The links could be declared in `application.yml` instead using `rel: reference` or `rel: guide` as appropriate. I think the link descriptions should reflect the titles of the pages to which they link.

Reply:
I will update them.

Reply:
When `spring-cloud-azure-starter-actuator`, `spring-cloud-azure-trace-sleuth`, `spring-cloud-azure-starter-integration-storage-queue` appear in the `pom`, the corresponding document cannot use `rel: reference` or `rel : guide` to add. This class should be needed.

Reply:
Thanks. I agree – for those dependencies that are added implicitly due to other dependencies having been selected we do need to do something. It looks like implicit dependencies ([as we now use for Testcontainers](https://github.com/spring-io/start.spring.io/commit/1cfafcf0b3ab761bcaddbffb779ab1f65867031d)) would be a good fit.

Reply:
Do think it makes sense to use implicit dependencies, @snicoll?

Reply:
Looks like it, yes!

Reply:
@wilkinsona. Does `SpringAzureHelpDocumentCustomizer` need to be changed? We just need to add one or more links in `HELP.md`.